core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -25,6 +25,7 @@ import scala.collection.JavaConverters._
 import org.apache.spark.{JobExecutionStatus, SparkConf}
 import org.apache.spark.scheduler.SparkListener
 import org.apache.spark.status.api.v1
+import org.apache.spark.status.api.v1.{ApplicationEnvironmentInfo, ApplicationInfo}
```

Comment:
Use existing `v1.blah` pattern.

Reply:
fixed
in addition removed some unused imports